src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -471,11 +471,18 @@ private void BeforeAttribute()
                 return;
             }
 
+            // Minimized attribute
             if (!At(HtmlSymbolType.Equals))
```

Comment:
Since parser gets confused by whitespace, how will `<element attribute = "value" />` or `<element attribute ="value" />` be parsed now?  Has this change made handling of (perfectly valid) whitespace before the equals _worse_?